prog/postgres/postgres_server_nexus.rb

```diff
@@ -357,6 +421,11 @@ def before_run
       hop_configure_prometheus
     end
 
+    when_configure_pgbouncer_set? do
```

Comment:
No one increments configure_pgbouncer, right? We actually want to set it up once at the initialization time. I think we can remove the semaphore and this check.

Reply:
I was wondering if this might come in handy for updating existing postgres servers to setup pgbouncer on them

Reply:
Since the number of pg instances is small, we'll do this manually as needed